halotools/mock_observables/counts_in_cells/tests/test_counts_in_cylinders.py

```diff
@@ -195,7 +195,7 @@ def test_counts_in_cylinders_error_handling():
         approx_cell1_size=0.2, approx_cell2_size=0.2)
 
 
-def test_cic_pbc():
```

Comment:
Changing these two made it easier to run all `counts_in_cylinders` tests in one go.